gmail/src/models/project.ts

```diff
@@ -0,0 +1,69 @@
+import { postJsonRpc } from "../utils/http";
+import { URLS } from "../const";
+import { ErrorMessage } from "../models/error_message";
+import { State } from "../models/state";
+
+/**
+ * Represent a "project.project" record.
+ */
+export class Project {
+    id: number;
+    name: string;
+
+    /**
+     * Unserialize the project object (reverse JSON.stringify).
+     */
+    static fromJson(values: any): Project {
+        const project = new Project();
+        project.id = values.id;
+        project.name = values.name;
+        return project;
+    }
+
+    /**
+     * Parse the dictionary return by the Odoo endpoint.
+     */
+    static fromOdooResponse(values: any): Project {
```

Comment:
Note that we also return company, in case we want to display it when in multi company mode. Even if we don't display company (maybe when having lot of projects / companies ? not sure), as we return it we could already store it.

Reply:
Yes, it's returned by the Odoo endpoint, but it's never used... It was originally to fix the multi-company issue, but NEB did it in a different way (add all users companies). Not sure it's a good idea to store it and never use it

Reply:
(note: in outlook we store it but never use it, should be probably cleaned)